content/en/agent/guide/autodiscovery-management.md

```diff
@@ -68,33 +51,33 @@ DD_AC_EXCLUDE = "image:dockercloud/network-daemon image:dockercloud/cleanup imag
 You can also use a regex to ignore them all: `DD_AC_EXCLUDE = "image:dockercloud/*"`
 
 {{% /tab %}}
-{{< /tabs >}}
-
-**Note**: If you are using Kubernetes, the container `<NAME>` is the one in your manifest `.spec.containers[0].name`.
-
-## Include Containers
-
-Include containers from the Agent Autodiscovery perimeter with an include rule based on their `name` or `image` to collect data **ONLY** from those containers. If a container matches an include rule, it's always included in the Autodiscovery perimeter.
-
-**Note**: Include rules support regexes, and are defined as a list of comma-separated strings.
-
-{{< tabs >}}
 {{% tab "Agent" %}}
 
-To include a given Docker container with the image `<IMAGE_NAME>` from Autodiscovery, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
+To remove a given Docker container with the image `<IMAGE_NAME>` from Autodiscovery, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
 
 ```yaml
-ac_include: [image:<IMAGE_NAME>]
+ac_exclude: [image:<IMAGE_NAME>]
 ```
 
-To include a given Docker container with the name `<NAME>` from Autodiscovery, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
+To remove a given Docker container with the name `<NAME>` from Autodiscovery, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
 
 ```yaml
-ac_include: [name:<NAME>]
+ac_exclude: [name:<NAME>]
 ```
 
 [1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
 {{% /tab %}}
+{{< /tabs >}}
+
+**Note**: If you are using Kubernetes, the container `<NAME>` is the one in your manifest `.spec.containers[0].name`.
+
+## Include Containers
+
+Include containers from the Agent Autodiscovery perimeter with an include rule based on their `name` or `image` to collect data **ONLY** from those containers. If a container matches an include rule, it's always included in the Autodiscovery perimeter.
```

Comment:
as FYI we are just about to add `kube_namespace` to AD_INCLUDE and AD_EXCLUDE, so it would be great to call it out here (should be available in 6.19) - https://github.com/DataDog/datadog-agent/pull/5224

Reply:
let's include this in another PR to avoid overloading this one